src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -14,6 +15,7 @@
 
 from WMCore.Services.PhEDEx.PhEDEx import PhEDEx
 from WMCore.Lexicon import cmsname, slicedIterator
+pnn_regex = re.compile(r'^T[0-3%]((_[A-Z]{2}(_[A-Za-z0-9]+)*)?)')
```

Comment:
@ericvaandering I have reverted this pattern to match the one for cmsname in https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Lexicon.py#L129

So that we can have these fixes in place I include it here as a module const until we think of a better approach. We can't use the method from Lexicon unless we want to be catching AssertionErrors everywhere., see https://github.com/alexanderrichards/WMCore/commit/a25af89d1eb6c1ac003672907db105a0d6b023ed#commitcomment-13953990 for the discussion